src/components/track.ts

```diff
@@ -89,6 +89,13 @@ export class Track extends ComponentBase {
 
     cloned.removeAttribute('id');
     cloned.classList.add('zodiac-cloned');
+    cloned.setAttribute('aria-hidden', 'true');
+
+    cloned.querySelectorAll('a').forEach((link) => {
```

Comment:
We should probably include selectors for all tabbable elements:
- `a`
- `button`
- `input`
- `select`
- `textarea`
- `[tabindex]`